next-app/src/components/views/Main/Top/Top.style.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components'
+import { colors } from 'styles/colors'
+import { getTypographyStyles } from 'styles/fonts'
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 300px;
+  background: ${colors.gray900};
+`
+export const Title = styled.h1`
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 48px;
+  color: ${colors.white};
+  text-align: center;
+  margin-bottom: 11px;
+`
+
+export const Subtitle = styled.h2`
+  ${getTypographyStyles('Headline4_M')}
+  color: ${colors.gray400};
+  text-align: center;
+`
+
+export const GoogleLoginButton = styled.button`
+  margin: 40px 0;
+  padding: 14px 50px;
+  border-radius: 30px;
+  background: ${colors.white};
+  cursor: pointer;
+`
```

Comment:
```suggestion
  ${getTypographyStyles('Headline4_B')};
`
```
여기에 타이포그라피 스타일 속성을 추가해주면 ButtonContentsWrapper와 p태그는 없앨 수 있을 것 같아요 👀

Reply:
불필요한 코드였네욤 ㅎㅎㅎ